qt/scientific_interfaces/ISISSANS/SANSRunWindow.cpp

```diff
@@ -886,8 +886,6 @@ bool SANSRunWindow::loadUserFile() {
     return false;
   }
 
-  const auto settings = getReductionSettings();
```

Comment:
Whilst it looks like the returned value does nothing the getReductionSettings function creates a PropertyManager object and stores it in the PropertyManagerDataService so I think this call may be needed anyway.

Reply:
I reverted the change and added a comment so that others will not burn their fingers here.